ppl/src/test/java/org/opensearch/sql/ppl/utils/PPLQueryDataAnonymizerTest.java

```diff
@@ -7,28 +7,56 @@
 package org.opensearch.sql.ppl.utils;
 
 import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
 import static org.opensearch.sql.ast.dsl.AstDSL.field;
 import static org.opensearch.sql.ast.dsl.AstDSL.projectWithArg;
 import static org.opensearch.sql.ast.dsl.AstDSL.relation;
 
+import com.google.common.collect.ImmutableSet;
 import java.util.Collections;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
 import org.opensearch.sql.ast.tree.UnresolvedPlan;
+import org.opensearch.sql.catalog.CatalogService;
 import org.opensearch.sql.ppl.antlr.PPLSyntaxParser;
 import org.opensearch.sql.ppl.parser.AstBuilder;
 import org.opensearch.sql.ppl.parser.AstExpressionBuilder;
 
+@RunWith(MockitoJUnitRunner.class)
 public class PPLQueryDataAnonymizerTest {
 
   private final PPLSyntaxParser parser = new PPLSyntaxParser();
 
+  @Mock
+  private CatalogService catalogService;
+
+  /**
+   * Setup the test context.
+   */
+  @Before
+  public void setUp() {
+    Mockito.lenient().when(catalogService.getCatalogs()).thenReturn(ImmutableSet.of());
+  }
+
   @Test
   public void testSearchCommand() {
     assertEquals("source=t | where a = ***",
```

Comment:
should it be `source=opensearch.t | where a = ***`

Reply:
I mean if there is no catalog mentioned, then it will default to opensearch. So not required as of now.